Build failure on android for version 7.0.0

Hi,

I'm getting the following build failure on android. PTAL. Thank you.

```
> Task :react-native-pdf:processReleaseManifest
package="org.wonday.pdf" found in source AndroidManifest.xml: /Users/amin/Projects/***/app/node_modules/react-native-pdf/android/src/main/AndroidManifest.xml.
Setting the namespace via the package attribute in the source AndroidManifest.xml is no longer supported, and the value is ignored.
Recommendation: remove package="org.wonday.pdf" from the source AndroidManifest.xml: /Users/amin/Projects/***/app/node_modules/react-native-pdf/android/src/main/AndroidManifest.xml.

> Task :react-native-pdf:compileReleaseJavaWithJavac FAILED
/Users/amin/Projects/***/app/node_modules/react-native-pdf/android/src/main/java/org/wonday/pdf/PdfManager.java:143: error: method setEnableRTL(PdfView,boolean) is already defined in class PdfManager
    public void setEnableRTL(PdfView view, boolean value) {
                ^
/Users/amin/Projects/***/app/node_modules/react-native-pdf/android/src/main/java/org/wonday/pdf/PdfView.java:294: error: cannot find symbol
                 ParcelFileDescriptor fileDescriptor = ParcelFileDescriptor.open(file, ParcelFileDescriptor.MODE_READ_ONLY);
                 ^
  symbol:   class ParcelFileDescriptor
  location: class PdfView
/Users/amin/Projects/***/app/node_modules/react-native-pdf/android/src/main/java/org/wonday/pdf/PdfView.java:294: error: cannot find symbol
                 ParcelFileDescriptor fileDescriptor = ParcelFileDescriptor.open(file, ParcelFileDescriptor.MODE_READ_ONLY);
                                                                                       ^
  symbol:   variable ParcelFileDescriptor
  location: class PdfView
/Users/amin/Projects/***/app/node_modules/react-native-pdf/android/src/main/java/org/wonday/pdf/PdfView.java:294: error: cannot find symbol
                 ParcelFileDescriptor fileDescriptor = ParcelFileDescriptor.open(file, ParcelFileDescriptor.MODE_READ_ONLY);
                                                       ^
  symbol:   variable ParcelFileDescriptor
  location: class PdfView
/Users/amin/Projects/***/app/node_modules/react-native-pdf/android/src/main/java/org/wonday/pdf/PdfView.java:295: error: cannot find symbol
                 PdfRenderer pdfRenderer = new PdfRenderer(fileDescriptor);
                 ^
  symbol:   class PdfRenderer
  location: class PdfView
/Users/amin/Projects/***/app/node_modules/react-native-pdf/android/src/main/java/org/wonday/pdf/PdfView.java:295: error: cannot find symbol
                 PdfRenderer pdfRenderer = new PdfRenderer(fileDescriptor);
                                               ^
  symbol:   class PdfRenderer
  location: class PdfView
/Users/amin/Projects/***/app/node_modules/react-native-pdf/android/src/main/java/org/wonday/pdf/PdfView.java:313: error: cannot find symbol
             } catch (IOException e) {
                      ^
  symbol:   class IOException
  location: class PdfView
/Users/amin/Projects/***/app/node_modules/react-native-pdf/android/src/main/java/org/wonday/pdf/PdfManager.java:29: error: PdfManager is not abstract and does not override abstract method setShowsVerticalScrollIndicator(PdfView,boolean) in RNPDFPdfViewManagerInterface
public class PdfManager extends SimpleViewManager<PdfView> implements RNPDFPdfViewManagerInterface<PdfView> {
       ^
Note: /Users/amin/Projects/***/app/node_modules/react-native-pdf/android/src/main/java/org/wonday/pdf/RNPDFPackage.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
8 errors

[Incubating] Problems report is available at: file:///Users/amin/Projects/***/app/android/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.
```

--- a/android/src/main/java/org/wonday/pdf/PdfManager.java
+++ b/android/src/main/java/org/wonday/pdf/PdfManager.java
@@ -99,6 +99,11 @@ public class PdfManager extends SimpleViewManager<PdfView> implements RNPDFPdfVi
         // NOOP on Android
     }
 
+	@Override
+    public void setShowsVerticalScrollIndicator(PdfView view, boolean value) {
+        // NOOP on Android
+    }
+
     @ReactProp(name = "enableRTL")
     public void setEnableRTL(PdfView view, boolean enableRTL) {
         pdfView.setEnableRTL(enableRTL);
@@ -137,11 +142,6 @@ public class PdfManager extends SimpleViewManager<PdfView> implements RNPDFPdfVi
     @ReactProp(name = "enablePaging")
     public void setEnablePaging(PdfView pdfView, boolean enablePaging) {
         pdfView.setEnablePaging(enablePaging);
-    }
-
-    @Override
-    public void setEnableRTL(PdfView view, boolean value) {
-        // NOOP on Android
     }
 
     @ReactProp(name = "fitPolicy")

--- a/android/src/main/java/org/wonday/pdf/PdfView.java
+++ b/android/src/main/java/org/wonday/pdf/PdfView.java
@@ -9,11 +9,13 @@
 package org.wonday.pdf;
 
 import java.io.File;
+import java.io.IOException;
 
 import android.content.ContentResolver;
 import android.content.Context;
 import android.os.Handler;
 import android.os.Looper;
+import android.os.ParcelFileDescriptor;
 import android.util.SizeF;
 import android.view.View;
 import android.view.ViewGroup;
@@ -22,7 +24,7 @@ import android.net.Uri;
 import android.util.AttributeSet;
 import android.view.MotionEvent;
 import android.graphics.Canvas;
-
+import android.graphics.pdf.PdfRenderer;
 
 import com.facebook.react.uimanager.ThemedReactContext;
 import com.facebook.react.uimanager.UIManagerHelper;

